tests/test_integrity.py

```diff
@@ -476,6 +477,51 @@ def test_sources(db):
     assert len(t) == 37, f'found {len(t)} sources from {ref}'
 
 
+# importing table 9 for data in test parameters
+from astropy.table import Table
+
+def test_ingest_modeledparameters(db):
```

Comment:
I think you got lucky that this ran before `test_modeledparameters`. I think maybe because it's alphabetically before? I don't know if pytest runs things sequentially or alphabetically, but if you want to guarantee that these inserts are in place before you run the tests either move them to `test_modeledparameters`, the `db` fixture at the start of the file, or as an extra fixture (probably not needed). 
Also, if you are inserting data, you could run into unrelated integrity errors if these sources are already present or the references aren't in place. Just a heads up to be clear on what you are testing.

Reply:
Thanks. I just deleted 481 as it's not actually needed. I had put it in a while ago and forgot